testnets/holesky/README.md

```diff
@@ -88,596 +90,601 @@ or **authorized delegates** acting on their behalf, to issue and sign preconfirm
 To learn more about delegation, check out the [Delegations and Signing](#delegations-and-signing-options-for-native-and-docker-compose-mode)
 section.
 
-# Off-Chain Setup
+> [!NOTE]
+> Before moving on to the actual instructions, please note that the on-chain steps must be completed before running the off-chain
+> infrastructure. The sidecar will verify that all of the associated validators and operator have been registered in the Bolt contracts,
+> else it will fail (for safety reasons).
 
-There are various way to run the Bolt Sidecar depending on what infrastructure
-you want to use and your preferred signing methods:
+# On-Chain Registration
 
-- Docker mode (recommended)
-- [Commit-Boost](https://commit-boost.github.io/commit-boost-client) mode
-  (requires Docker)
-- Native mode (advanced, requires building everything from source)
+Once you are successfully running the Bolt sidecar you need to register on-chain
+on the Bolt Registry to successfully receive preconfirmation requests from users
+and RPCs. This step is needed to provide economic security to your
+commitments.
```

Comment:
I think this paragraph isn't consistent with the previous callout: you're previously asking to run the on-chain steps first, but then you say _"Once you are successfully running the Bolt sidecar..."_ which suggests you should run the sidecar first.

Wdyt?

Reply:
True, will take another look at that